zcash_client_sqlite/CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ and this library adheres to Rust's notion of
 ## [Unreleased]
 ### Changed
 - MSRV is now 1.70.0.
+- `zcash_client_sqlite::error::SqliteClientError` has changed variants:
+  - Removed `HdwalletError`.
+  - Added `TransparentDerivation`.
```

Comment:
Note that we also currently do an `.unwrap()` after calling `derive_secret_key` in `hdwallet`, in `zcash_client_backend` here:
https://github.com/zcash/librustzcash/blob/03fc64cfd03547483d448f60797545ce3ae5b147/zcash_client_backend/src/data_api/wallet.rs#L826-L829

It would be good to be able to report a useful error there, but `zcash_client_backend` has no error type to do so.